rpmlint/checks/FileDigestCheck.py

```diff
@@ -34,46 +38,90 @@ def _get_digest_group_from_path(self, path):
                     pass
         return None
 
-    def check_binary(self, pkg):
+    def _in_digest_configuration_group(self, pkgfile):
+        if stat.S_ISDIR(pkgfile.mode):
+            return False
+        return self._get_digest_location_from_path(pkgfile.name)
```

Comment:
Since this function seems to want to return a boolean value, adding `is not None` might be a good idea here.

Reply:
Yes.